pages/index.tsx

```diff
@@ -1,61 +1,67 @@
+import {useState} from 'react'
 import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import ipsumWords from '../data/ipsumWords'
 
 export default function Home() {
+	const [paragraphCount, setParagraphCount] = useState(1)
+	const [wordCount, setWordCount] = useState(ipsumWords.length)
+
+	// Creates a paragraph array with word limit
+	const paragraph = ipsumWords.slice(0, wordCount).join(' ')
+
+	// Generates lorum ipsum paragraph elements
+	// Array.from(Array(paragraphCount)) creates an empty iterable array 
+	const generate = Array.from(Array(paragraphCount)).map((par, idx) => {
+		return(
```

Comment:
spacing? it looks like a tab instead of spaces. is prettier and autoformat not working?